cmd/config/config.go

```diff
@@ -21,6 +22,12 @@ type SMTP struct {
 	AuthPassword string `mapstructure:"auth_password" validate:"required"`
 }
 
+type AWS struct {
+	AccessKey       string `mapstructure:"access-key" validate:"required"`
+	SecretAccessKey string `mapstructure:"secret-access-key" validate:"required"`
+	Region          string `mapstructure:"region" validate:"required"`
+}
+
```

Comment:
What if we use multiple AWS services with different region?

Reply:
we wont 😂

Reply:
even so, ideally the access should be granted by CLI through bash scripts, not on the App layer
hence the solution, but this is just a simple one cause setting up the script might introduce some unnecessary complexity for our repo